pins_data_model/gen_models.py

```diff
@@ -3,15 +3,13 @@
 """
 
 import json
-from os import path
 from pathlib import Path
 from datamodel_code_generator import InputFileType, PythonVersion, generate
 from datamodel_code_generator import DataModelType
 
 from load_schemas import load_all_schemas
 
-modelsDir = path.join(path.dirname(path.realpath(__file__)), "models")
-
+models_dir = Path.cwd() / "models"
```

Comment:
I don't think this is quite equivalent, if the script isn't run in this directory.

Do we need `Path.cwd()` -> `Path(__file__).resolve()` ?